src/libs/notifications/notifications.js

```diff
@@ -0,0 +1,28 @@
+import { navigateToApp } from '/libs/functions/openApp'
+import { generateWebLink } from 'cozy-client'
+
+const navigateFromNotification = (client, navigate, notification) => {
```

Comment:
Could be awesome to create a type for the notification object

Reply:
I tried to convert notifications.js in ts in a new commit, tell me what you think @Crash-- @acezard